README.md

```diff
@@ -11,53 +11,238 @@ Pour faire ce codelab, vous avez besoin des outils suivants :
 * NodeJS 8.x

```

Comment:
Can we provide a link for the users that don't use to work with node for easy integration ?

Reply:
I agree